res/sections/20-reading.tex

```diff
@@ -0,0 +1,68 @@
+% Eliminare colonne vuote
+% Verificare che variabili qualitative lo siano davvero
+\chapter{Leggere correttamente i dati}
+
+A volte può capitare che i dati che abbiamo non siano completi, o che ci sia la
+necessità di verificare che alcune variabili abbiano le proprietà desiderate.
+Per fare ciò R mette a disposizione dei semplici comandi che verranno spiegati
+di seguito.
+
+\section{Filtrare dati letti}
+
+Questa funzionalità è particolarmente utile quanto ci ritroviamo con dei dati
+che possono contenere delle celle ad esempio vuote o con dati che sappiamo non
+essere corretti.
+
+Il comando per vedere se ci sono dati vuoti in un \textit{dataset} è
+\texttt{is.na()}, dove, come parametro della funzione, dobbiamo passare i
+nostri dati. In aggiunta a questo comando possiamo aggiungere \texttt{sum()},
+che permette di contare vari elementi. Combinando \texttt{sum()} con
+\texttt{is.na()} avremo il numero di celle che contengono dati vuoti.
+Proviamo con un esempio:
+
+\input{res/Rnw/20-reading1.Rnw.include.generated}
+
+Per esempio con il \textit{dataset} R ci dice che ci sono $59$ dati nulli.
+Come possiamo eliminare queste celle vuote? Un metodo per farlo è usando la
+seguente funzione:
+
+\input{res/Rnw/20-reading7.Rnw.include.generated}
+
+Questa funzione restituisce il dataset che gli passiamo come argomento, ma
+senza i dati vuoti. Se vogliamo salvare il tutto dobbiamo eseguire
+l'assegnazione ad una nuova variabile, e per fare ciò dobbiamo usare il simbolo
+di assegnazione, che in R è \texttt{<-}.
+
+\section{Variabili quantitative e qualitative}
+
+Con la statistica ci ha insegnato (o avrebbe dovuto) che le variabili possono
+considerarsi di due tipi: qualitative o quantitative. R è in grado di eseguire
+questa distinzione tra le variabili, ma a volte potrebbe sbagliare.
```

Comment:
Specificherei che può sbagliare nel momento in cui si caricano dati da file e questo è un problema se si crea un modello più che altro

Reply:
Va bene se metto la roba del modello in un footnote?